src/mon/MonCommands.h

```diff
@@ -235,6 +235,10 @@ COMMAND("report name=tags,type=CephString,n=N,req=false", \
 	"mon", "r", "cli,rest")
 COMMAND("quorum_status", "report status of monitor quorum", \
 	"mon", "r", "cli,rest")
+COMMAND("mon_metadata name=rank,type=CephInt,range=0",
```

Comment:
thank you @jcsp . yes, that's actually what i came up first. but i changed it to "mon_metadata" later on. 

because, "$foo $bar" command are handled by module named `$foo` where $foo can be mds,fs,osd,pg,mon,auth ... . in the case of mon, the `$bar` command will be handled by `MonmapMonitor` which is a PaxosService, but the implementation of `mon_metadata` lives in `Monitor` itself. and a plain Dispatcher will serve my implementation well as suggested by @jecluis . maybe i can rework the way how Monitor dispatches commands so we are not limited by this constraint?

what do you think?